evansmusomi/react-native-102
# Adds Signup Elements on Auth Screen and Styling

*What this does*
- Adds sign up input fields and button on Auth screen
- Adds global text input and text styles
- Adds background image on Auth screen

*How to verify*
- Navigate to Auth screen
- You should see a background image
- The "Please log in" should appear as a header in bold, black, larger font size
- The input fields should have a white background and grey borders
